testcases/PowerNVDump.py

```diff
@@ -643,6 +647,10 @@ def runTest(self):
 
 class KernelCrash_KdumpNetwork(PowerNVDump):
 
+    # This test verifies kdump/fadump over ssh ans nfs.
```

Comment:
over ssh `s/ans/and` nfs.

Reply:
or can be re worded as 
`This test verifies kdump/fadump over network ( ssh and nfs).`